ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3ClientBuilder_Test.java

```diff
@@ -31,6 +32,27 @@ public void createBasicClient() throws Exception {
     }
 
     @Test
+    public void createBasicClientFromENV() throws Exception {
+        try {
+            final Ds3ClientBuilder builder = Ds3ClientBuilder.fromEnv();
```

Comment:
Right, so I set up to pass if the variables aren't setup--basically, the test is only valid if the environment variables are set up.